tests/test_hypothesis.py

```diff
@@ -77,7 +78,7 @@ def test_equality(d):
 def test_consistency_after_mutation(d, arg, itemlist):
     for arity, mms in mutating_methods_by_arity.items():
         for mm in mms:
-            b = bidict(d)
+            b = orderedbidict(d) if 'orderedbidict' in repr(mm) else bidict(d)
```

Comment:
Since unbound methods went away in python 3, I resorted to this to check whether the class `mm` came from was `bidict` or `orderedbidict` (see http://stackoverflow.com/a/25959545 for why). I think this is fine for the purposes of this test, but open to suggestions otherwise.